demos/mpu9250-i2c/Cargo.toml

```diff
@@ -9,6 +9,7 @@ description = "Demonstration of interfacing an MPU9250 with the Teensy 4"
 
 [dependencies]
 cortex-m = "0.6.3"
+cortex-m-rt = "0.6.13"
```

Comment:
Instead of that hack (above), we can directly use this library to boot the processor.